src/aks-preview/azext_aks_preview/_loadbalancer.py

```diff
@@ -26,48 +26,65 @@ def set_load_balancer_sku(sku, kubernetes_version):
+def create_load_balancer_profile(managed_outbound_ip_count, managed_outbound_ipv6_count, outbound_ips,
+                                 outbound_ip_prefixes, outbound_ports, idle_timeout):
     """parse and build load balancer profile"""
-    if not is_load_balancer_profile_provided(managed_outbound_ip_count, outbound_ips, outbound_ip_prefixes,
-                                             outbound_ports, idle_timeout):
+    if not is_load_balancer_profile_provided(managed_outbound_ip_count, managed_outbound_ipv6_count, outbound_ips,
+                                             outbound_ip_prefixes, outbound_ports, idle_timeout):
         return None
 
     profile = ManagedClusterLoadBalancerProfile()
-    return configure_load_balancer_profile(managed_outbound_ip_count, outbound_ips, outbound_ip_prefixes,
-                                           outbound_ports, idle_timeout, profile)
+    return configure_load_balancer_profile(managed_outbound_ip_count, managed_outbound_ipv6_count, outbound_ips,
+                                           outbound_ip_prefixes, outbound_ports, idle_timeout, profile)
 
 
-def configure_load_balancer_profile(managed_outbound_ip_count, outbound_ips, outbound_ip_prefixes, outbound_ports,
-                                    idle_timeout, profile):
+def configure_load_balancer_profile(managed_outbound_ip_count, managed_outbound_ipv6_count, outbound_ips,
+                                    outbound_ip_prefixes, outbound_ports, idle_timeout, profile):
     """configure a load balancer with customer supplied values"""
     if not profile:
         return profile
 
     outbound_ip_resources = _get_load_balancer_outbound_ips(outbound_ips)
     outbound_ip_prefix_resources = _get_load_balancer_outbound_ip_prefixes(outbound_ip_prefixes)
 
-    if managed_outbound_ip_count or outbound_ip_resources or outbound_ip_prefix_resources:
+    if (managed_outbound_ip_count or managed_outbound_ipv6_count or
+            outbound_ip_resources or outbound_ip_prefix_resources):
         # ips -> i_ps due to track 2 naming issue
         profile.managed_outbound_i_ps = None
         # ips -> i_ps due to track 2 naming issue
         profile.outbound_i_ps = None
         profile.outbound_ip_prefixes = None
+        if managed_outbound_ipv6_count:
+            if profile.managed_outbound_i_ps:
+                profile.managed_outbound_i_ps.count_ipv6 = managed_outbound_ipv6_count
+            else:
+                # ips -> i_ps due to track 2 naming issue
+                profile.managed_outbound_i_ps = ManagedClusterLoadBalancerProfileManagedOutboundIPs(
+                    count_ipv6=managed_outbound_ipv6_count
+                )
         if managed_outbound_ip_count:
             # ips -> i_ps due to track 2 naming issue
             profile.managed_outbound_i_ps = ManagedClusterLoadBalancerProfileManagedOutboundIPs(
                 count=managed_outbound_ip_count
             )
```

Comment:
The value (`count_ipv6`) might be overwritten when user set `--load-balancer-managed-outbound-ip-count` and `--load-balancer-managed-outbound-ipv6-count` at the same time. I am not sure if this is expected.